test/test_ldap_connection.rb

```diff
@@ -291,7 +291,7 @@ def test_queued_read_setup_encryption_with_start_tls
       and_return(result2)
     mock.should_receive(:write)
     conn = Net::LDAP::Connection.new(:socket => mock)
-    flexmock(Net::LDAP::Connection).should_receive(:wrap_with_ssl).with(mock, {}).
+    flexmock(Net::LDAP::Connection).should_receive(:wrap_with_ssl).with(mock, {}, nil).
```

Comment:
I tested the change by setting a very low timeout on LDAPS connection and
make sure an exception is raised.
On Mon, Jun 20, 2016 at 10:15 PM Tatsuya Sato <notifications@github.com>
wrote:

> In test/test_ldap_connection.rb
> <https://github.com/ruby-ldap/ruby-net-ldap/pull/273#discussion_r67809160>
> :
>
> > @@ -291,7 +291,7 @@ def test_queued_read_setup_encryption_with_start_tls
> >        and_return(result2)
> >      mock.should_receive(:write)
> >      conn = Net::LDAP::Connection.new(:socket => mock)
> > -    flexmock(Net::LDAP::Connection).should_receive(:wrap_with_ssl).with(mock, {}).
> > +    flexmock(Net::LDAP::Connection).should_receive(:wrap_with_ssl).with(mock, {}, nil).
>
> @tmm1 <https://github.com/tmm1> Doesn't this mocking cover your changes
> on wrap_with_ssl, do ti? If you tested your changes work as you expected,
> could you share how to do it?
>
> I know it's really hard to reproduce the situation timeout or something
> network issues happens.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ruby-ldap/ruby-net-ldap/pull/273/files/749c22b4e5514ead10c92bcaec1c5a1eb49db455#r67809160>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAKB01CFZ0ewSejsl4qpinzGhwx48U5ks5qN3N4gaJpZM4I25cW>
> .
>